src/components/Person/Person.jsx

```diff
@@ -1 +1,38 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => (
```

Comment:
You can destructure props here and you won`t need 'person.' everywhere
```suggestion
export const Person = ({ person }) => {
  const {
    name,
    age,
    sex,
    isMarried,
    partnerName,
  } = person;
```

Reply:
agree